ooniprobe/View/RunTestViewController.mm

```diff
@@ -0,0 +1,121 @@
+//
+//  RunTestTableViewController.m
+//  ooniprobe
+//
+//  Created by Lorenzo Primiterra on 06/08/17.
+//  Copyright © 2017 Simone Basso. All rights reserved.
+//
+
+#import "RunTestViewController.h"
+
+@interface RunTestViewController ()
+
+@end
+
+@implementation RunTestViewController
+@synthesize testName, testArguments, testDecription;
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(reloadTestUI) name:@"reloadTable" object:nil];
+
+    currentTests = [Tests currentTests];
+    [self.runButton setTitle:[NSString stringWithFormat:@"   %@   ", NSLocalizedString(@"run", nil)] forState:UIControlStateNormal];
+
+    if (testDecription != nil)
```

Comment:
This is conditionally set using a `synthetized` setter in the code above, correct?

Reply:
I don't get your comment

Reply:
Nevermind; beware that there is a typo: it should be `testDescription` not `testDecription`